src/components/PeopleTable.tsx

```diff
@@ -1,644 +1,126 @@
+import { Person } from '../types';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { SearchLink } from '../components/SearchLink';
+import { PersonLink } from '../PersonLink/PersonLink';
+import classNames from 'classnames';
+
 /* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+  const [searchParams] = useSearchParams();
+
+  const isSelected = people.find(person => person.slug === slug);
+
+  const currentSort = searchParams.get('sort');
+  const currentOrder = searchParams.get('order');
+
+  const getNextSortParams = (field: string) => {
+    if (currentSort === field && currentOrder === 'asc') {
+      return { sort: field, order: 'desc' };
+    } else if (currentSort === field && currentOrder === 'desc') {
+      return { sort: null, order: null };
```

Comment:
Returning `{ sort: null, order: null }` might not effectively clear the sorting parameters from the URL. Consider omitting these parameters entirely or using an empty string to ensure they are removed.